src/styles/block/card.scss

```diff
@@ -0,0 +1,108 @@
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  align-items: center;
```

Comment:
Define `box-sizing: border-box` always like this
```
* {
box-sizing: border-box
}
```
cause usually we want this behaviour for all elements